vendor/README.md

```diff
@@ -378,6 +378,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **SystemVerilog:** [bitbucket:Clams/sublimesystemverilog](https://bitbucket.org/Clams/sublimesystemverilog)
 - **TLA:** [agentultra/TLAGrammar](https://github.com/agentultra/TLAGrammar)
 - **TOML:** [textmate/toml.tmbundle](https://github.com/textmate/toml.tmbundle)
+- **TSQL:** [textmate/sql.tmbundle](https://github.com/textmate/sql.tmbundle)
```

Comment:
There was a collision here when I attempted to add a different grammar for TSQL since sql.tmbundle already used up `source.sql` and the one I was adding also tried to add `source.sql`. If there is a way around that, I can try adding the grammar again.

Reply:
If a grammar supports multiple languages or variations of the language, as it appears this grammar might, then it's fine to reuse the same grammar.

Reply:
That said, I'm not 100% sure this grammar supports TSQL based on https://github.com/textmate/sql.tmbundle/issues/25 - it also appears to be unmaintained, so if you've got a better source that covers all the languages this grammar is currently used for, and it has support for TSQL, feel free to swap it in this PR.

Reply:
You are correct, and looking at files I know to be TSQL on GitHub, I can see some syntax highlighting issues. The problem I have is, I am not sure if there is a grammar that supports all of them. I checked the 2 you mentioned in the Issue, and the vscode one is at least still maintained, but looks like it works exclusively for TSQL, and the other one is just a little less stale than sql.tmbundle and likely only works for TSQL as well.

Reply:
If you can't find a grammar that covers all, the other option would be to submit a PR or issue to one of the still maintained grammars suggesting changing the scope to something like `source.tsql`.  https://github.com/kinnell/atom-language-transact-sql (I think this is a TSQL grammar) looks like it's quite well maintained so this might be a good place to start.

You can test the grammar using lightshow as referenced in the PR template.

Reply:
Lightshow does not appear to work well with the one you suggested (they use a .plist to defined the pattern, but use .less to manage coloring, I don't know if that would be a problem if it were to be used here). I will continue to poke around.

Reply:
This grammar (https://github.com/theia-ide/theia/blob/master/packages/textmate-grammars/data/sql.tmLanguage.json) seems to work really well for TSQL. Is it possible to add just that as the grammar? Or no?

Reply:
Nope. The grammar's in the wrong path for Linguist to pick it up. You can however use the original source - https://github.com/Microsoft/vscode-mssql/ - which would probably be a better choice as it's actively maintained. The only snag is it also uses the `source.sql` scope so will need to support all the other SQLs too 😞

Reply:
Darn, the reason I liked the Theia one better is because it actually works through Lightshow. The Microsoft one (https://github.com/Microsoft/vscode-mssql/blob/master/syntaxes/SQL.plist) does not.